code/_globalvars/configuration.dm

```diff
@@ -2,7 +2,6 @@ var/datum/configuration/config = null
 
 var/host = null
 var/join_motd = null
-var/station_name = "NSS Cyberiad"
```

Comment:
Isn't this the string that comes after "Paradise Station -- " on the byond hub? Did you make sure that this doesn't break the byond hub entry?

Reply:
You are right, and really this should trigger the station name, rather than be hard coded.

Reply:
I did break it, thanks for the heads up. Should be fixed now.